mmv1/third_party/terraform/services/dataplex/resource_dataplex_glossary_category_test.go.tmpl

```diff
@@ -0,0 +1,92 @@
+package dataplex_test
+
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"testing"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccDataplexGlossaryCategory_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataplexGlossaryCategory_update(context),
+			},
+			{
+				ResourceName:            "google_dataplex_glossary_category.category_test_id",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"category_id", "glossary_id", "labels", "location", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccDataplexGlossaryCategory_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_dataplex_glossary" "category_test_id" {
```

Comment:
Replace all instances of `category_test_id` with `category_test_id_full` so that`testAccDataplexGlossaryCategory_update()` and  `testAccDataplexGlossaryCategory_dataplexGlossaryCategoryFull()` are referring to the same resource.